lib/nexmo_developer/spec/services/translator/file_translator_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Translator::FileTranslator do
+  let(:doc_path) { 'voice/voice-api/guides/numbers.md' }
+  subject { described_class.new(doc_path) }
+
+  it 'accepts a documentation path to initialize an instance' do
+    expect(subject.doc_path).to eql(doc_path)
+  end
+
+  describe '#load_frontmatter' do
+    it 'loads the document metadata correctly' do
+      expect(subject.load_frontmatter).to include('translation_frequency' => 10)
+    end
+  end
+
+  describe '#translation_requests' do
+    it 'returns an array of Translator::TranslationRequest instances' do
+      requests = subject.translation_requests
+
+      expect(requests).to all(be_a(Translator::TranslationRequest))
+    end
+  end
+
+  describe '#find_matching_product' do
+    it 'finds the matching product translation frequency in the products config YAML file' do
+      product = subject.find_matching_product
+
+      expect(product).to eql(15)
+    end
```

Comment:
can we add a test for when the `translation_frequency` is missing?

Reply:
👍